consensus/block_validation.go

```diff
@@ -0,0 +1,69 @@
+package consensus
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+type BlockValidator interface {
+	BlockSemanticValidator
+	BlockSyntaxValidator
+}
+
+type BlockSemanticValidator interface {
+	ValidateSemantic(ctx context.Context, child, parent *types.Block) error
+}
+
+type BlockSyntaxValidator interface {
+	ValidateSyntax(ctx context.Context, blk *types.Block) error
+}
+
+type BlockValidationClock interface {
+	Now() int64
+	BlockTime() time.Duration
```

Comment:
This is unused right now and boils down to a constant. I suggest removing it from the interface and making it a direct member of the DefaultBlockValidator.

The clock interface will then be more widely useable (later): perhaps rename `Now` to `EpochSeconds` for extensibility.